crates/net/downloaders/src/headers/linear.rs

```diff
@@ -883,7 +872,7 @@ mod tests {
     use reth_interfaces::test_utils::{TestConsensus, TestHeadersClient};
     use reth_primitives::SealedHeader;
 
-    static CONSENSUS: Lazy<Arc<TestConsensus>> = Lazy::new(|| Arc::new(TestConsensus::default()));
+    static CONSENSUS: Lazy<Arc<dyn Consensus>> = Lazy::new(|| Arc::new(TestConsensus::default()));
```

Comment:
we should get rid of this static in tests altogether

Reply:
good idea, could do that in a follow up